src/aura/STG_CMP_Addr/STG_CMP_AddrHelper.js

```diff
@@ -0,0 +1,17 @@
+({
+	runBackfill : function (component, event, helper) {
+		// console.log();
+		var runBatchAction = component.get("c.executeEthnicityRaceBatch");
+		runBatchAction.setCallback(this, function(response) {
+	        if(response.getState() === "SUCCESS") {
+
+			    var backfillBtn = component.find("ethnicRaceMsg");
+			    $A.util.toggleClass(backfillBtn, "slds-hide");
+
+	        } else if(response.getState() === "ERROR") {
+	        	this.displayError(response);
```

Comment:
Good catch here. I tested this and received an error but it was not actually the correct error after double checking based on your response. There was a problem with `displayError()` all along. It will be fixed when my clean up is committed.